src/xGadgetron/pGadgetron/tests/CMakeLists.txt

```diff
@@ -1,6 +1,4 @@
 #========================================================================
-# Author: Edoardo Pasca
-# Copyright 2017 University College London
 # Copyright 2017 Science Technology Facilities Council
 # Author: Casper da Costa-Luis
 # Copyright 2017
```

Comment:
now that I've started (?) to be annoying, checking the history, this needs to be 2017-2021. I'd remove Casper and the 2017 below as his change here is half a line (of course, the whole thing is rather trivial and copyright statements for this type is file is a bit silly)